docker_images/sklearn/app/pipelines/common.py

```diff
@@ -37,21 +38,28 @@ def __init__(self, model_id: str):
                 config = json.load(f)
         except Exception:
             config = dict()
-            raise UserWarning("`config.json` does not exist or is invalid.")
+            warnings.warn("`config.json` does not exist or is invalid.")
 
         self.model_file = (
             config.get("sklearn", {}).get("model", {}).get("file", DEFAULT_FILENAME)
         )
+        self.model_format = (
+            config.get("sklearn", {}).get("model_format", "pickle")
```

Comment:
FYI this assumes file is pickle because of how there's no skops models without config (the ones without config are to maintain vanilla sklearn models that weren't uploaded with skops)